Assets/HoloToolkit/Utilities/Editor/ProjectSettingsWindow.cs

```diff
@@ -0,0 +1,274 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+	/// <summary>
+	/// Renders the UI and handles update logic for HoloToolkit/Configure/Apply HoloLens Project Settings.
+	/// </summary>
+	public class ProjectSettingsWindow : AutoConfigureWindow<ProjectSettingsWindow.ProjectSetting>
+	{
+		// Nested Types
+		public enum ProjectSetting
+		{
+			ActiveBuildToWsa,
+			WsaSdkToUwp,
+			WsaUwpBuildToD3D,
+			WsaFastestQuality,
+			WsaEnableVR
+		}
+
+		// Private Methods
+
+		/// <summary>
+		/// Enables virtual reality for WSA and ensures HoloLens is in the supported SDKs.
+		/// </summary>
+		private void EnableVirtualReality()
+		{
+			try
+			{
+				// Grab the text from the project settings asset file
+				string settingsPath = "ProjectSettings/ProjectSettings.asset";
+				string settings = File.ReadAllText(settingsPath);
+
+				// We're looking for the list of VR devices for the current build target, then
+				// ensuring that the HoloLens is in that list
+				bool foundBuildTargetVRSettings = false;
+				bool foundBuildTargetMetro = false;
+				bool foundBuildTargetEnabled = false;
+				bool foundDevices = false;
+				bool foundHoloLens = false;
+
+				var builder = new StringBuilder(); // Used to build the final output
```

Comment:
I agree explicit declaration does make the code more readable. Thanks Jared for updating!